HPXMLtoOpenStudio/resources/EPvalidator.xml

```diff
@@ -0,0 +1,980 @@
+    <!-- [Neighbors] -->
+    <sch:rule context="/h:HPXML/h:Building/h:BuildingDetails/h:BuildingSummary/h:Site/h:extension/h:Neighbors">
+      <sch:assert test="count(h:NeighborBuilding) &gt;= 1">Expected 1 or more element(s): NeighborBuilding</sch:assert> <!-- See [NeighborBuilding] -->
+    </sch:rule>
+  </sch:pattern>
+
+  <sch:pattern name="[NeighborBuilding]">
+    <!-- [NeighborBuilding] -->
+    <sch:rule context="/h:HPXML/h:Building/h:BuildingDetails/h:BuildingSummary/h:Site/h:extension/h:Neighbors/h:NeighborBuilding">
+      <sch:assert test="count(h:Azimuth) = 1">Expected 1 element(s): Azimuth</sch:assert>
+      <sch:assert test="count(h:Distance) = 1">Expected 1 element(s): Distance</sch:assert> <!-- ft -->
+      <sch:assert test="count(h:Height) &lt;= 1">Expected 0 or 1 element(s): Height</sch:assert> <!-- ft; if omitted, the neighbor is the same height as the main building -->
+    </sch:rule>
+  </sch:pattern>
+
+  <sch:pattern name="[VentedAttic]">
+    <!-- [VentedAttic] -->
+    <sch:rule context='/h:HPXML/h:Building/h:BuildingDetails/h:Enclosure/h:Roofs/h:Roof[h:InteriorAdjacentTo="attic - vented"]'>
+      <sch:assert test="count(../../h:Attics/h:Attic[h:AtticType/h:Attic[h:Vented='true']]/h:VentilationRate[h:UnitofMeasure='SLA' or h:UnitofMeasure='ACHnatural']/h:Value) &lt;= 1">Expected 0 or 1 element(s): ../../Attics/Attic[AtticType/Attic[Vented="true"]]/VentilationRate[UnitofMeasure="SLA" or UnitofMeasure="ACHnatural"]/Value</sch:assert>
+    </sch:rule>
+  </sch:pattern>
+  
+  <sch:pattern name="[VentedCrawlspace]">
+    <!-- [VentedCrawlspace] -->
+    <sch:rule context='/h:HPXML/h:Building/h:BuildingDetails/h:Enclosure/h:FoundationWalls/h:FoundationWall[h:InteriorAdjacentTo="crawlspace - vented"]'>
+      <sch:assert test="count(../../h:Foundations/h:Foundation[h:FoundationType/h:Crawlspace[h:Vented='true']]/h:VentilationRate[h:UnitofMeasure='SLA']/h:Value) &lt;= 1">Expected 0 or 1 element(s): ../../Foundations/Foundation[FoundationType/Crawlspace[Vented="true"]]/VentilationRate[UnitofMeasure="SLA"]/Value</sch:assert>
+    </sch:rule>
+  </sch:pattern>
+
+  <sch:pattern name="[FoundationWallInsLayer]">
+    <!-- [FoundationWallInsLayer] -->
+    <sch:rule context='/h:HPXML/h:Building/h:BuildingDetails/h:Enclosure/h:FoundationWalls/h:FoundationWall/h:Insulation/h:Layer[h:InstallationType="continuous - exterior" or h:InstallationType="continuous - interior"]'>
+      <sch:assert test="count(h:NominalRValue) = 1">Expected 1 element(s): NominalRValue</sch:assert>
+      <sch:assert test="count(h:extension/h:DistanceToTopOfInsulation) = 1">Expected 1 element(s): extension/DistanceToTopOfInsulation</sch:assert> <!-- ft -->
+      <sch:assert test="count(h:extension/h:DistanceToBottomOfInsulation) = 1">Expected 1 element(s): extension/DistanceToBottomOfInsulation</sch:assert> <!-- ft -->
+    </sch:rule>
+  </sch:pattern>
+
+  <sch:pattern name="[FrameFloorAdjacentToOther]">
+    <!-- [FrameFloorAdjacentToOther] -->
```

Comment:
Seems like we can remove all of these comments since it's identical to the pattern name on the line above it.